docs/source/bond_family_guide/design/object_model/quote.rst

```diff
@@ -0,0 +1,111 @@
+   or licensors. Title to the Material remains with Intel Corporation or its
+   suppliers and licensors. The Material contains trade secrets and proprietary
+   and confidential information of Intel or its suppliers and licensors. The
+   Material is protected by worldwide copyright and trade secret laws and
+   treaty provisions. No part of the Material may be used, copied, reproduced,
+   modified, published, uploaded, posted, transmitted, distributed, or
+   disclosed in any way without Intel's prior express written permission.
+
+   No license under any patent, copyright, trade secret or other intellectual
+   property right is granted to or conferred upon you by disclosure or delivery
+   of the Materials, either expressly, by implication, inducement, estoppel or
+   otherwise. Any license under such intellectual property rights must be
+   express and approved by Intel(R) in writing.
+
+.. _object_type_quote:
+
+object-type: quote
+==================
+
+.. todo::
+
+   For object-type quote:
+     - Is the definition of the firm attribute correct?
+     - More information on the format of ask-price and bid-price.
+
+
+Attributes
+----------
+
+  object-id
+    See :ref:`common_attributes`.
+
+  object-type
+    The string 'quote'.
+
+  creator-id
+    See :ref:`common_attributes`.
+
+  firm
+    The pricing source of the organization associated with this quote.
+
+  cusip
+    An identifier for the bond.
+
+    See :ref:`object_type_bond`.
+
+    Optional (one of ison or cusip must be set)
+
+  isin
+    And identifier for this bond.
+
+    See :ref:`object_type_bond`.
+
+    See: https://en.wikipedia.org/wiki/International_Securities_Identification_Number
+
+    Optional (one of ison or cusip must be set)
+
+  ask-price
+    The minimum price for which the firm is willing to sell the bond.
+
+  ask-qty
+    The maximum quantity the firm is willing to sell.
+
+  bid-price
+    The maximum price the firm is willing to pay to buy this bond.
+
+  bid-qty
+    The maximum quantity the firm is willing to buy.
+
+  timestamp
+    The time that the quote is created in relation to the current clock.
+
+  status
+    One of 'Open' or 'Closed'.
+
+    Quotes are 'Open' if they have sufficient remaining buy and
+    sell quantities and have not been explicitly cancelled.
+
+    Quotes move to 'Closed' status if they can no longer fulfill
+    a round-lot order (buy or sell quantity < 100000) or if they
+    are explicitly cancelled by an authorized member of the firm
+    associated with the quote.
+
+Example JSON
+------------
+
+.. code-block:: json
+
+   {
+       "object-id" : "OBJECT_ID",
+       "object-type" : "quote",
+       "creator-id": "OBJECT_ID",
+       "firm" : "SCGV",
+       "isin" : "US00206RDA77",
+       "ask-price" : "101-06 7/8",
+       "ask-qty" : 1000000,
+       "bid-price" : "101-05 7/8",
+       "bid-qty" : 1000000,
+       "timestamp" : 1469045984.965841,
```

Comment:
The float form of the timestamp is correct. The quote timestamp is set here: https://github.com/hyperledger/sawtooth-core/blob/0-7/extensions/bond/sawtooth_bond/updates/settlement.py#L721, but the quote gets it from the latest ClockUpdate which is set here: https://github.com/hyperledger/sawtooth-core/blob/0-7/extensions/bond/sawtooth_bond/txn_family.py#L88. time.time() gives seconds (in float) since Unix epoch.